site/components/header/small-screen-nav.js

```diff
@@ -14,24 +15,45 @@ export default class SmallScreenNav extends React.Component {
   }
 
   handleInputChange = event => {
+    this.scrollLock();
```

Comment:
I assume this is ok but it would be good to make sure @samsmith1983 is aware of the scroll handler stuff so he can test perf on all devices.